src/components/ArticleCard/ArticleCard.spec.js

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import ArticleCard from './index';
+
+describe('ArticleCard', () => {
+  let articleCard;
+  const title = 'Pointing';
+  const description =
+    'Things are going to be fine. All you need to do is stay focused and believe in yourself';
+  const body =
+    'The journey to become world class may be scary and daunting. All you need to do is stay focused and believe in yourself';
+  const image = 'image';
+  const authorImage = 'author-image';
+  const createdAt = 'June 25';
+  const author = 'Mark';
```

Comment:
Good mocked article data here @tobechukwuobitube . However I think, this article  mock data can be moved to the mockData file in our project rather than have it cluster about. and I'd also suggest you bring the different littered variables into an object.
Example:
```js
const articleCardMockData = {title: 'Pointing', description:.....}
```

Reply:
Thanks a lot, @kechyy. 👍